src/main/java/org/galatea/starter/domain/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package org.galatea.starter.domain.exception;
+
+public class EntityNotFoundException extends RuntimeException {
```

Comment:
This exception is likely to be used only in the RestController so I think it should not be in the domain package, but closer to where it will actually be used.